src/platform/forms-system/test/js/state/helpers.unit.spec.js

```diff
@@ -1,10 +1,12 @@
 import { expect } from 'chai';
+import sinon from 'sinon';
```

Comment:
All the changes in this file are also done as a smaller PR here https://github.com/department-of-veterans-affairs/vets-website/pull/11617